maxim/models/evaluator.py

```
@@ -106,9 +106,9 @@ def dict_to_class(cls, data: Dict[str, Any]) -> "LocalEvaluatorReturn":
 @dataclass
 class PassFailCriteriaOnEachEntry:
     score_should_be: OperatorType
-    value: Union[bool, int]
+    value: Union[bool, int, float,None]
 
-    def __init__(self, score_should_be: OperatorType, value: Union[bool, int]):
+    def __init__(self, score_should_be: OperatorType, value: Union[bool, int, float]):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)